support_model.py

```diff
@@ -44,6 +45,17 @@ def remove_slack_commands(query):
     return query
 
 
+def get_answer_if_known(query):
+    gc = gspread.service_account(filename='ods-qna-dd4a610895ad.json')
+    questions = gc.open("faq fixed").sheet1
+
+    list_of_dicts = questions.get_all_records()
+
+    for dct in list_of_dicts:
```

Comment:
Unclear name. What dicts? Please, name it as business entities

Reply:
- [ ] rename